ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -83,9 +99,10 @@ public sealed class ObservableAsPropertyHelper<T> : IHandleObservableErrors, IDi
 
             _lastValue = initialValue;
             _source = observable.StartWith(initialValue).DistinctUntilChanged().Multicast(subj);
-            if (ModeDetector.InUnitTestRunner())
+            if (subscribeImmediately || ModeDetector.InUnitTestRunner())
             {
                 _inner = _source.Connect();
+                _activated = 1;
```

Comment:
@kentcb We can add this line, even if we don't merge this PR, so connect isn't called a second time in `Value`